README.md

```diff
@@ -18,12 +18,14 @@
   - [Project repo options that has s2i templates and Jenkins Pipelines](#project-repo-options-that-has-s2i-templates-and-jenkins-pipelines)
     - [OpenShift s2i template setup options](#openshift-s2i-template-setup-options)
     - [Jenkins 2.0 pipeline setup options](#jenkins-20-pipeline-setup-options)
+  - [Templates](#templates)
   - [Usage examples](#usage-examples)
-    - [Example 1: Setup on a local server :: Setup Minishift + OS templates](#example-1-setup-on-a-local-server--setup-minishift--os-templates)
-    - [Example 2: Setup on a local server :: Setup Minishift + OS templates + Jenkins 2.0 pipelines](#example-2-setup-on-a-local-server--setup-minishift--os-templates--jenkins-20-pipelines)
-    - [Example 3: Setup on a local server :: Setup Minishift + OS templates + Jenkins 2.0 pipelines](#example-3-setup-on-a-local-server--setup-minishift--os-templates--jenkins-20-pipelines)
+    - [Example 1: Setup on a local machine :: Setup Minishift + OS templates](#example-1--setup-on-a-local-machine----setup-minishift-os-templates)
```

Comment:
Did you use doctoc to generate a new TOC?

Reply:
@arilivigni no, I edit using vim.

Reply:
please use markdown-toclify to generate the TOC.